Makefile

```diff
@@ -12,7 +12,7 @@ all:
 	go test -v ./...
 	mkdir -p bin/
 	go build  -o bin/getcreds  -ldflags "-X main.Version=${VERSION}" cmd/getcreds/main.go
-	go build  -o bin/keymaster -ldflags "-X main.Version=${VERSION}" cmd/ssh_usercert_gen/main.go cmd/ssh_usercert_gen/templateData.go
+	go build  -o bin/keymaster -ldflags "-X main.Version=${VERSION}" cmd/ssh_usercert_gen/main.go cmd/ssh_usercert_gen/auth_oauth2.go cmd/ssh_usercert_gen/templateData.go
```

Comment:
Why are you doing it this way? The convention is to have Go compile all the (non-test) code in a directory.